cmake/uny-compilerflags.cmake

```diff
@@ -40,4 +40,34 @@ if (SANITIZE)
 
     # UndefinedBehaviorSanitizer is a fast undefined behavior detector.
     set(UNY_DEBUG_FLAGS "${UNY_DEBUG_FLAGS} -fsanitize=undefined")
+endif()
+
+# shared CXX flags for src & tests
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${UNY_FLAGS}")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${UNY_DEBUG_FLAGS}")
+SET(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} ${UNY_LINKER_FLAGS}")
```

Comment:
These are fine here, but should we probably start using:

set_target_properties($TARGET PROPERTIES
    COMPILE_FLAGS "${UNY_FLAGS}"
    LINK_FLAGS "${UNY_LINKER_FLAGS}"
)

?